jupyterhub_singleuser_profiles/images.py

```diff
@@ -47,6 +48,14 @@ def get_default(self):
                 return image.name
 
         return image_list[0].name if len(image_list) else None
+    
+    def get_image_build_status(self, imagestream_name):
+        build_list = self.openshift.get_builds()
+        for build in build_list.items:
+            #if imagestream_name in build.metadata.name:
+            if imagestream_name == build.spec.output.to.name:
```

Comment:
Is this the optimal way to gather the build belonging to an imagestream?

Reply:
maybe @vpavlin could give you more insight

Reply:
Yeah, I think that's fine